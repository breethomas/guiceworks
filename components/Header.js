// @flow
import React from 'react'

export default (props: any) => (
  <header {...props} />
)
