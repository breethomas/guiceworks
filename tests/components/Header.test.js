import React from 'react'
import renderer from 'react-test-renderer'
import Header from '../../components/Header'

describe('Header', () => {
  let component

  beforeEach(() => {
    component = <Header />
  })

  it('shows the correct snapshot for the Header component tree', () => {
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
