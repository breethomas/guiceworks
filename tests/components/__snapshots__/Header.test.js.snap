// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header shows the correct snapshot for the Header component tree 1`] = `<header />`;
